### ISSUE TITLE
MultiBody UsersGuide

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#534]
**Reported by anonymous on 21 Apr 2011 08:53 UTC**
The MultiBody UsersGuide states the following announcement

```
Note, in this Beta-release, not all provided line force
components have already an optional mass. This will be fixed
in the next release.
```

This announcement can already be found in all relases of the MSL 3.x. The question is which MSL version will finally contain the announced fixes, if ever?

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/534

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 21 Apr 2011 19:06 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by anonymous on 5 Aug 2011 11:43 UTC**
Can you please change milestone to MSL 3.3.

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 22 Aug 2011 11:17 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by anonymous on 1 Nov 2012 12:08 UTC**
Martin, it is still there. Please remove the release prediction and fix this announcement to something like: Note, not all provided line force components have an optional mass.

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 9 Dec 2012 15:51 UTC**
Fixed in 824368072b4738681df731a41e741fa9e512dbdf by removing the comment

**Comment by modelica-trac-importer** [comment]
**Changelog removed by otter on 9 Dec 2012 15:51 UTC**
